pkgs/tools/misc/bc/default.nix

```diff
@@ -7,11 +7,22 @@ stdenv.mkDerivation rec {
     sha256 = "62adfca89b0a1c0164c2cdca59ca210c1d44c3ffc46daf9931cf4942664cb02a";
   };
 
-  configureFlags = [ "--with-readline" ];
+  configureFlags = [
+    "--with-readline"
+    "CC_FOR_BUILD=${buildPackages.stdenv.cc.targetPrefix}gcc"
```

Comment:
Should be done by stdenv and by default.

Reply:
Agreed, although actually I think we can omit this for now unless someone has a config where buildPackages' targetPrefix is not empty?

(pushed variant that drops this for now)